### PR TITLE
Adding Document Shortcuts to Mac-keyboard.json

### PR DESCRIPTION
**Pull request by AcriCAA**
**What does your Instant Answer do?**

Provides document keyboard shortcuts (especially paste and match style) 

**What is the data source for your Instant Answer? (Provide a link if possible)**

https://support.apple.com/en-us/HT201236

**Why did you choose this data source?**

Apple support doc. 

**Are there any other alternative (better) data sources?**

No. 

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Mac users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

I modified an existing cheatsheet. 

**Are you having any problems? Do you need our help with anything?**
I only want to submit a pull request for my latest commit but don't know how to separate the previous two commits

**Where did you hear about DuckDuckHack? (For first time contributors)**
Zac 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screen shot 2015-08-05 at 12 14 07 pm

### CODE REVIEW

**Comment by moollaza** [comment]
@AcriCAA thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: lib/DDG/Goodie/IsAwesome/AcriCAA.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package DDG::Goodie::IsAwesome::AcriCAA;
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/AcriCAA.pm]: Seeing as this is a new Instant Answer, it needs to be in its own PR. Can you please remove it from this PR?

File: share/goodie/cheat_sheets/mac-keyboard.json [hunk 2]
```diff
@@ -62,7 +62,11 @@
         }, {
             "key" : "[Cmd] + [↑/↓]",
             "val" : "Skip to next/previous result category"
-        }], "Text Editing" :[{
+        }],
+
```
Comment (moollaza) [inline share/goodie/cheat_sheets/mac-keyboard.json]: Let's reduce this to a single newline?

File: share/goodie/cheat_sheets/mac-keyboard.json [hunk 3]
```diff
@@ -148,10 +152,258 @@
...
+        }],
+        
+      "Document shortcuts" : [
+
+            {
+                "key": "[Cmd] + [Shift] + [Opt] + [V]",
+                "val": "Paste and Match Style: Apply the style of the surrounding content to the item pasted within that content."
+            },
+          
+               {
```
Comment (moollaza) [inline share/goodie/cheat_sheets/mac-keyboard.json]: nit: please fix the indentation here

File: share/goodie/cheat_sheets/mac-keyboard.json [hunk 4]
```diff
@@ -148,10 +152,258 @@
...
         }, {
             "key" : "[Cmd] + [F]",
             "val" : "Find"
-        }]
+        }],
+        
+      "Document shortcuts" : [
+
+            {
+                "key": "[Cmd] + [Shift] + [Opt] + [V]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/mac-keyboard.json]: Please remove all the `+`'s from the JSON. We changed the syntax a month or so ago and we've opted not to include them anymore.

File: share/goodie/cheat_sheets/mac-keyboard.json [hunk 5]
```diff
@@ -148,10 +152,258 @@
...
+            {
+                "key": "[Ctrl] + [O]",
+                "val": "Insert a new line after the insertion point."
+            },
+            {
+                "key": "[Ctrl] + [T]",
+                "val": "Swap the character behind the insertion point with the character in front of the insertion point."
+            },
+            {
+                "key": "[Cmd] + [Left Curly Bracket ({)]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/mac-keyboard.json]: I don't think we need to give both the name and character. The latter should be enough. Also, don't forget to escape it!

```json
"key": "[Cmd] [\\{]"
```

File: share/goodie/cheat_sheets/mac-keyboard.json [hunk 6]
```diff
@@ -148,10 +152,258 @@
...
+            {
+                "key": "[Cmd] + [Shift] + [P]",
+                "val": "Page setup: Display a window for selecting document settings."
+            },
+            {
+                "key": "[Cmd] + [Shift] + [S]",
+                "val": "Display the Save As dialog, or duplicate the current document."
+            },
+            {
+                "key": "[Cmd] + [Shift] +   [Minus sign ( - )]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/mac-keyboard.json]: Same here, let's just use the character. We wrap these in code blocks so it's clear to users what they're supposed to press.

File: lib/DDG/Goodie/IsAwesome/AcriCAA.pm (hunk 1 above)
Comment (AcriCAA) [inline lib/DDG/Goodie/IsAwesome/AcriCAA.pm]: I don't know how.
